### ISSUE TITLE
Shell evasion by $IFS / ${IFS}

### ISSUE THREAD

**Issue by lifeforms** [SpiderLabs/owasp-modsecurity-crs#664]
In bash, `$IFS` can be used as a replacement for the space character. This trick is actively used.

Example:

```
GET /language/Swedish${IFS}&&echo${IFS}610cker>qt&&tar${IFS}/string.js HTTP/1.0
Accept: */*
User-Agent: Wget(linux)
```

I propose to add `$IFS` and some [interesting other variables](http://www.tldp.org/LDP/abs/html/internalvariables.html) to `unix-shell.data` and also add REQUEST_URI to scanned variables.

The expression `${IFS}` is already caught by rule 932130, but to catch this example, we should also add REQUEST_URI to scanned variables.

**Comment by dune73** [comment]
Good catch. I was not aware of this possibility of `$IFS`.

What other variables do you have in mind?

I am a bit reluctant to extend an existing rule with REQUEST_URI for a point release. So are you aiming for 3.0.1 or would you wait with that for 3.1.0

**Comment by csanders-git** [comment]
interesting note. I'm glad you shared it.

**Comment by lifeforms** [comment]
Implemented in #1325
